README.md

```diff
@@ -133,6 +135,10 @@ For any details on Cert-Manager, check the [official documentation](https://cert
   kubectl get secret example-com-tls
   ```
 
-### Contributing
+## Contributing
+
+Check [CONTRIBUTING.md](CONTRIBUTING.md).
+
+## Security
 
-Kms-Issuer is built using the [Kubebuilder](https://book.kubebuilder.io/) framework. See the [official documentation](https://book.kubebuilder.io/quick-start.html) to get started.
```

Comment:
I think we need to keep this stanza.

Reply:
it was moved to contributor's guide (dependencies section). Do you think it should be put back in `README`?